model/cloud.go

```diff
@@ -30,3 +30,28 @@ type StripeSetupIntent struct {
 type ConfirmPaymentMethodRequest struct {
 	StripeSetupIntentID string `json:"stripe_setup_intent_id"`
 }
+
+// Subscription model represents a subscription on the system.
+type Subscription struct {
+	ID                 string   `json:"id"`
+	CustomerID         string   `json:"customer_id"`
+	ProductID          string   `json:"product_id"`
+	AddOns             []string `json:"add_ons"`
+	StartAt            int64    `json:"start_at"`
+	EndAt              int64    `json:"end_at"`
+	CreateAt           int64    `json:"create_at"`
+	Seats              int      `json:"seats"`
+	DNS                string   `json:"dns"`
+	StripeClientSecret string   `json:"stripe_client_secret,omitempty"`
+	LicensePayload     string   `json:"license_payload"`
```

Comment:
I would omit these two fields give that the `LicensePayload` does not apply for cloud subscriptions and `StripeClientSecret` isn't needed in the frontend either. 

I'm debating as well on `DNS` and `Seats`. For cloud subscriptions they don't purchase an amount of seats instead they are billing on monthly usage.